docs/reference/release-notes/8.0.0-beta1.asciidoc

```diff
@@ -13,9 +13,9 @@ Also see <<breaking-changes-8.0,Breaking changes in 8.0>>.
 === Known issues
 
 * If you're using {ml}, it's not safe to perform a rolling upgrade from `7.16.x`
-  to `8.0.0-beta1`. Doing so might lead to cluster state propagation issues in the mixed 
-  version cluster. We recommend waiting for `8.0.0-rc1` before testing a 
-  rolling upgrade from `7.16.x`. Do not attempt to upgrade any 
+  to `8.0.0-beta1`. Doing so might lead to cluster state propagation issues in the mixed
+  version cluster. We recommend waiting for `8.0.0-rc1` before testing a
+  rolling upgrade from `7.16.x`. Do not attempt to upgrade any
   production cluster to pre-release software.
```

Comment:
The way this bullet reads currently it feels like if I'm using ML I can't try out 8.0.0-beta1 at all. This would obviously limit the ML teams feedback from the beta. I think it might be worth adding that these users can upgrade to 8.0.0-beta1 if they use a full cluster restart upgrade (assuming thats true)

Reply:
I agree, something like this could help to clarify:

>  We recommend performing a full cluster restart upgrade, or waiting for `8.0.0-rc1`...

Reply:
Let's tag in @droberts195 for this question, since I did not write that item and don't know the answer myself. I think it is a good discussion, but outside the scope of my PR.

Reply:
I removed that whole bullet point in https://github.com/elastic/elasticsearch/pull/80417 because the underlying problem got fixed when 8.0.0-beta1 got rebuilt. So hopefully there should be no confusion now.

Reply:
@williamrandolph I just merged latest master into this PR and now the bullet point that generated this conversation is gone from what's visible in this PR review.

Reply:
Thanks!